packages/mask/src/components/shared/ApplicationBoard.tsx

```diff
@@ -350,52 +367,63 @@ export function ApplicationBoard({ secondEntries, secondEntryChainTabs }: MaskAp
-                        setChainId={setChainId}
-                        classes={classes}
-                        chains={secondEntryChainTabs}
+        <TargetChainIdContext.Provider>
+            <AllProviderTradeContext.Provider>
+                {secondEntryChainTabs?.length ? (
+                    <div className={classes.abstractTabWrapper}>
+                        <NetworkTab
+                            chainId={chainId}
+                            setChainId={setChainId}
+                            classes={classes}
+                            chains={secondEntryChainTabs}
+                        />
+                    </div>
+                ) : null}
+                <section className={classes.applicationWrapper}>
+                    {(secondEntries ?? firstLevelEntries).map(
+                        ({ title, img, onClick, supportedChains, hidden, walletRequired }, i) =>
+                            (!supportedChains || supportedChains?.includes(chainId)) && !hidden ? (
+                                <div
+                                    className={classNames(
+                                        classes.applicationBox,
+                                        walletRequired && !selectedWallet ? classes.disabled : '',
+                                    )}
+                                    onClick={onClick}
+                                    key={i}>
+                                    <img src={img} className={classes.applicationImg} />
+                                    <Typography className={classes.title} color="textPrimary">
+                                        {title}
+                                    </Typography>
+                                </div>
+                            ) : null,
+                    )}
+                </section>
+                {isClaimAllDialogOpen ? (
+                    <ClaimAllDialog open={isClaimAllDialogOpen} onClose={onClaimAllDialogClose} />
+                ) : null}
+
+                {isSecondLevelEntryDialogOpen ? (
+                    <EntrySecondLevelDialog
+                        title={secondLevelEntryDialogTitle}
+                        open={isSecondLevelEntryDialogOpen}
+                        entries={secondLevelEntries}
+                        chains={secondLevelEntryChains}
+                        closeDialog={onSecondLevelEntryDialogClose}
+                    />
+                ) : null}
+                {isFindTrumanDialogOpen ? (
+                    <FindTrumanDialog open={isFindTrumanDialogOpen} onClose={onFindTrumanDialogClose} />
+                ) : null}
+
+                {isSwapDialogOpen ? <TraderDialog open={isSwapDialogOpen} onClose={onSwapDialogClose} /> : null}
+                {isSavingsDialogOpen ? (
+                    <SavingsDialog
+                        open={isSavingsDialogOpen}
+                        onClose={onSavingsDialogClose}
+                        onSwapDialogOpen={onSwapDialogOpen}
```

Comment:
Since these dialogs are in the true branch of the ternary expression, you can omit the `isXxxx`.

```suggestion
                {isClaimAllDialogOpen ? (
                    <ClaimAllDialog open onClose={onClaimAllDialogClose} />
                ) : null}

                {isSecondLevelEntryDialogOpen ? (
                    <EntrySecondLevelDialog
                        title={secondLevelEntryDialogTitle}
                        open
                        entries={secondLevelEntries}
                        chains={secondLevelEntryChains}
                        closeDialog={onSecondLevelEntryDialogClose}
                    />
                ) : null}
                {isFindTrumanDialogOpen ? (
                    <FindTrumanDialog open onClose={onFindTrumanDialogClose} />
                ) : null}

                {isSwapDialogOpen ? <TraderDialog onClose={onSwapDialogClose} /> : null}
                {isSavingsDialogOpen ? (
                    <SavingsDialog
                        open
                        onClose={onSavingsDialogClose}
                        onSwapDialogOpen={onSwapDialogOpen}
```

Or you can just write the dialogs without ternary expression
```suggestion

                    <ClaimAllDialog open={isClaimAllDialogOpen} onClose={onClaimAllDialogClose} />

                    <EntrySecondLevelDialog
                        title={secondLevelEntryDialogTitle}
                        open={isSecondLevelEntryDialogOpen}
                        entries={secondLevelEntries}
                        chains={secondLevelEntryChains}
                        closeDialog={onSecondLevelEntryDialogClose}
                    />


                    <FindTrumanDialog open={isFindTrumanDialogOpen} onClose={onFindTrumanDialogClose} />


                 <TraderDialog open={isSwapDialogOpen} onClose={onSwapDialogClose} /> 
                    <SavingsDialog
                        open={isSavingsDialogOpen}
                        onClose={onSavingsDialogClose}
                        onSwapDialogOpen={onSwapDialogOpen}
```